channeldb/channel.go

```diff
@@ -2111,6 +2111,13 @@ func (c *OpenChannel) AppendRemoteCommitChain(diff *CommitDiff) error {
 			return ErrChanBorked
 		}
 
```

Comment:
Was there a commit missing before this one? It starts with "In the same vein, ..." as if there was some prior commit that it's referencing context from.

Reply:
I think so, will modify commit message